### PR TITLE
Add test cases for user dictionary 

### LINKED ISSUE #59: Update of user dictionary
We tried to clean up user dictionary to avoid custom tokenization because found some error in it after data was loaded into index. However Elastic was not able to start at all. Look like empty file scenario is not supported.

### PR DESCRIPTION
**Pull request by johtani**
Add test cases for user dictionary
Fix wrong property name of user dictionary

Closes #59

### CODE REVIEW

**Comment by johtani** [comment]
Now, we fix only master branch, because Lucene fix it on 5x and trunk.
I think fixing it to 1.5 and 1.x is helpful.
If fixing it to 1.5, I should add catch block for NullPointerException to 
https://github.com/elastic/elasticsearch-analysis-kuromoji/blob/a3dce667b18c1587b6392f44faa957b863cd521c/src/main/java/org/elasticsearch/index/analysis/KuromojiTokenizerFactory.java#L63

File: src/test/java/org/elasticsearch/index/analysis/KuromojiAnalysisTests.java [hunk 1]
```diff
@@ -245,4 +245,28 @@ private String readFully(Reader reader) throws IOException {
...
+        Tokenizer tokenizer = tokenizerFactory.create();
+        tokenizer.setReader(new StringReader(source));
+        assertSimpleTSOutput(tokenizer, expected);
+    }
+
+    // fix #59
+    @Test
+    public void testKuromojiEmptyUserDict() {
+        AnalysisService analysisService = createAnalysisService();
+        try {
```
Comment (dadoonet) [inline src/test/java/org/elasticsearch/index/analysis/KuromojiAnalysisTests.java]: You probably don't need to try catch here.
If an exception is thrown the test will fail

**Comment by dadoonet** [comment]
Left a small comment.
LGTM

File: src/test/java/org/elasticsearch/index/analysis/KuromojiAnalysisTests.java (hunk 1 above)
Comment (johtani) [inline src/test/java/org/elasticsearch/index/analysis/KuromojiAnalysisTests.java]: I added try catch to make it clear that does not throw exception.

**Comment by johtani** [comment]
thanks, @dadoonet !

**Pull request closed without merging**